### PR TITLE
Feat : Final Order Page On Live

### PR DESCRIPTION
**Pull request by Rishi05051997**
Peer Members, please review Order page

### CODE REVIEW

File: src/styles/order.css [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+.header-container {
+    grid-column:header;
+}
+
+.order-section {
+    grid-column:order-section;
+    grid-column: order-section;
+    margin: 30rem 10rem;
+    /* height: 100vh; */
```
Comment (parulgupta2001) [inline src/styles/order.css]: remove commented code

**Review by parulgupta2001 — APPROVED** [review]
apply the suggestion  otherwise everything looks ok

**Review by maitrakhatri — APPROVED** [review]
Clean code.
Suggestion: if the features are very small, merger 2-3 or more features and raise a combined PR

File: src/styles/order.css (hunk 1 above)
Comment (Rishi05051997) [reply src/styles/order.css]: Ok Done

**Pull request merged**